### PR TITLE
fixes issue where override datasources url is just a string not an object

### PR DESCRIPTION
**Pull request by mattmediavine**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -64,8 +64,15 @@ export const withPgAdapter = <PrismaClientType>(
     },
   })
 
-  const { url } = prisma._engineConfig.overrideDatasources?.db ?? prisma._engineConfig.inlineDatasources.db
-  const pool = new Pool({ connectionString: url.value || process.env[url.fromEnvVar] });
+  let connectionString;
+  if (prisma._engineConfig.overrideDatasources?.db?.url) {
+    connectionString = prisma._engineConfig.db.url;
```
Comment (exAspArk) [inline src/index.ts]: Hey @mattmediavine — should this be `prisma._engineConfig.overrideDatasources.db.url`?

Comment (mattmediavine) [reply src/index.ts]: Nope, fat fingered. I noticed this also and already made a change

Comment (mattmediavine) [reply src/index.ts]: It should be one optional as overrideDatasources may not have a db property if it is not being used.

Comment (exAspArk) [reply src/index.ts]: Sorry, to clarify, it seems like `*.overrideDatasources.*` is missing in this chain?

Comment (mattmediavine) [reply src/index.ts]: you were absolutely correct. Sorry I have fixed that issue.

Comment (exAspArk) [reply src/index.ts]: Thank you! 🙌 

This should be released in `v0.8.1`

**Pull request merged**